src/managers/DataManager.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const BaseManager = require('./BaseManager');
+
+let Structures;
+
+/**
+ * Manager that holds data structures.
```

Comment:
This feels misleading, it doesn't necessarily hold these data structures, they seem to be anything that accesses data held in a different cache, presenting it in a different (limited) way.

Reply:
Docs could be clearer yea, but this isn't just for those weird ones. `CachedManager` extends this, too.

Reply:
But...that has its own description (also that says it "holds a cache" which...feels wrong too.

Reply:
Does this sound better?

Reply:
yeah, if people actually care about what these managers do (since each manager that extends them has its own description) they should be able to understand that.